src/analyzer/trackanalysisscheduler.cpp

```diff
@@ -63,7 +63,10 @@ TrackAnalysisScheduler::TrackAnalysisScheduler(
         kLogger.debug()
                 << "Starting"
                 << numWorkerThreads
-                << "worker threads";
+                << "worker threads. Ideal: "
+                << QThread::idealThreadCount()
```

Comment:
Mentioning the ideal thread count doesn't matter here after a specific number of worker threads has been requested. Please note that we already start 2 different schedulers with different numbers of worker threads.

Printing the ideal thread count at this point is confusing.